scripts/Frequency_Domain_Analysis.py

```diff
@@ -0,0 +1,29 @@
+#pylint: disable=invalid-name
+from __future__ import (absolute_import, division, print_function)
+from Muon import FFT_presenter
+from Muon import FFT_view
+from PyQt4.QtGui import *
```

Comment:
Can we avoid `import *`, please? I think `from PyQt4.QtGui import QApplication, QMainWindow` will suffice here.